src/Entities/Child.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace Choredo\Entities;
+
+use Choredo\Entities\Behaviors;
+use Doctrine\ORM\Mapping as ORM;
+use Ramsey\Uuid\UuidInterface;
+
+/**
+ * @package Choredo\Entities
+ *
+ * @ORM\Entity(repositoryClass="Choredo\Repositories\ChildRepository")
+ * @ORM\Table(name="children")
+ * @ORM\HasLifeCycleCallbacks
+ */
+class Child
+{
+    use Behaviors\HasUuid;
+    use Behaviors\HasCreatedDate;
+    use Behaviors\HasUpdatedDate;
+    use Behaviors\BelongsToFamily;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string")
+     */
+    private $name;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string", name="access_code")
+     */
+    private $accessCode;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string", name="avatar_uri")
+     */
+    private $avatarUri;
+
+    /**
+     * Child constructor.
+     * @param UuidInterface $id
+     * @param string $name
+     * @param string $accessCode
+     * @param string $avatarUri
+     * @param Family $family
+     */
+    public function __construct(UuidInterface $id, string $name, string $accessCode, string $avatarUri, Family $family)
```

Comment:
It might be worth restructuring models so family comes first, or family after ID. 

It didn't make much difference in `Account`, but in the case of Child (and probably others), there will be nullable fields and Family can't go last. I'd rather see it first in all models, since putting it last in all models isn't possible and putting it at a random place in the argument string seems weird. Thoughts?